include/unifex/await_transform.hpp

```diff
@@ -73,7 +74,12 @@ class _as_awaitable {
     
     void set_done() && noexcept {
       op_->state_ = state::done;
-      continuation_.resume();
+      if constexpr (remove_cvref_t<Sender>::sends_done) {
```

Comment:
We should stop looking for these things directly on the `Sender` type and instead use `sender_traits`.

Reply:
I've fixed this - and uncommented the `sender_traits<S>::sends_done` definition so that it's usable ;)